apps/server/src/infra/etherpad-client/loggable/etherpad-server-error-exception.ts

```diff
@@ -0,0 +1,28 @@
+import { HttpExceptionOptions, InternalServerErrorException } from '@nestjs/common';
+import { ErrorLogMessage, Loggable } from '@src/core/logger';
+import { ErrorType, EtherpadParams } from '../interface';
+
+export class EtherpadServerError extends InternalServerErrorException implements Loggable {
```

Comment:
I guess, according to our naming scheme this should be named "EtherpadErrorLoggableException" or something like that.

Reply:
Rename class.